plugins/octopus-deploy/src/api/index.ts

```diff
@@ -156,6 +164,30 @@ export class OctopusDeployClient implements OctopusDeployApi {
     return responseJson;
   }
 
+  async getProjectGroups(): Promise<OctopusProjectGroup[]> {
+    const proxyUrl = await this.discoveryApi.getBaseUrl('proxy');
+    const url = `${proxyUrl}${this.proxyPathBase}/projectgroups/all`;
+    const response = await this.fetchApi.fetch(url);
```

Comment:
Wondering if it might make sense to abstract this into a private function similar to `getProjectApiUrl`, but for project groups (or adjust the current abstraction to support project groups urls).

Reply:
I've updated the implementation a bit. Maybe later we can look at using the Typescript Octopus Api package created by octopus itself [here](https://github.com/OctopusDeploy/api-client.ts)